segmentation_models_pytorch/encoders/timm_universal.py

```diff
@@ -36,3 +48,11 @@ def out_channels(self):
     @property
     def output_stride(self):
         return min(self._output_stride, 2**self._depth)
+
+
+def _merge_kwargs_no_dupls(a: dict[str, Any], b: dict[str, Any]) -> dict[str, Any]:
+    dupls = a.keys() & b.keys()
+    if dupls:
+        raise ValueError(f"'{dupls}' already specified internally")
+
+    return a | b
```

Comment:
Let's refactor it a bit:
1) dupls -> duplicates
2) "|" not sure which python version introduced it, maybe `{**a, **b}` would be more robust

Reply:
2. Looks like it was added in Python 3.9, so no need to change this: https://peps.python.org/pep-0584/